lib/phoenix_mtm/helpers.ex

```diff
@@ -50,6 +52,20 @@ defmodule PhoenixMTM.Helpers do
     )
   end
 
+  defp build_label_with_input(form, field, input_tag, label, label_opts, [nested: true]) do
+    [
+      label form, field, label_opts do
+        [{:safe, "#{label}"}, input_tag]
```

Comment:
I was concerned that a `safe` tuple would be skipped when using `html_escape` but just tested it and it's fine.

Let's leave it as is.